docs/source/azure/policy/resources/arm.rst

```diff
@@ -10,6 +10,13 @@ Filters
 - ARM Resource Filters (see :ref:`azure_genericarmfilter`)
     - Tag Filter - Filter on tag presence and/or values
     - Marked-For-Op Filter - Filter on tag that indicates a scheduled operation for a resource
+- `resource-type` filter
+    - Filter on ARM resource types, including resource type namespaces
+    - Provide a list of resource types (case-insensitive) in a `values` array
```

Comment:
fwiw, I have recently realized that single ticks are almost never what you actually want in reStructuredText.  Inline code is is a **double** back tick.  Probably the two here just render like normal text.   The single ticks are code to be evaluated I think, like for references.  

https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html